CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* linux-yocto: Include support for the RealSense cmaera [Theodor]
```

Comment:
typo
> cmaera

Also, the Changelog does not need to have per recipe details, so no need to prefix it with `linux-yocto`